src/fr/charly/javaTutorial/JavaLesson2.java

```diff
@@ -0,0 +1,19 @@
+package fr.charly.javaTutorial;
+
+public class JavaLesson2 {
+
+	public static void main(String[] args) {
+		
+		int money = 850;
+		int pricephone = 800;
+		boolean hasPhone = false;
+		
+		if(money >= pricephone && !hasPhone){
+			System.out.println("tu peut acheter ce telephone");
+			} else {
```

Comment:
The code is not well indented.
Use the "Ctrl + Shift + F" to indent it correctly.